src/rf/apps/home/views.py

```diff
@@ -205,3 +205,25 @@ def _get_layers(request, crit=None):
     """
     models = _get_layer_models(request, crit)
     return [m.to_json() for m in models]
+
+
+@api_view
+@accepts('POST')
+@csrf_exempt
+def update_status(request):
+    """
+    Update processing status on a Layer.
+    """
+    if request.method == 'POST':
+        layer_id = request.POST['layer_id']
```

Comment:
I thought each Layer is only processed once, so it doesn't make sense to also have a job_id. I could be wrong about that though.